transform/utils/utils.py

```diff
@@ -108,3 +110,17 @@ def extract_digits_and_trim(input_str: str):
     remaining_str = input_str[index:].strip()
 
     return remaining_str
+
+
+def map_currency_symbol(currency_symbol):
+    """Maps a currency symbol to its corresponding value."""
+    return currency_mapping.get(currency_symbol, currency_symbol)
```

Comment:
its so short that it shouldn't be a function. Just use it directly in the code

Reply:
The function has been relocated to the ProjectTransformer class and now includes a warning that verifies if the value is present in the dictionary keys or values.